server/lomas_server/utils/config.py

```diff
@@ -34,37 +36,55 @@ class Server(BaseModel):
 class DatasetStoreConfig(BaseModel):
     """BaseModel for dataset store configs"""
```

Comment:
would be now better to rename as BasicDatasetStoreConfig and update the docstring, right?